substrate/frame/election-provider-multi-phase/src/mock.rs

```diff
@@ -441,7 +441,7 @@ where
 	type Extrinsic = Extrinsic;
 }
 
-pub type Extrinsic = sp_runtime::testing::TestXt<RuntimeCall, ()>;
+pub type Extrinsic = sp_runtime::generic::UncheckedExtrinsic<u64, RuntimeCall, (), ()>;
```

Comment:
Can this be backwards compatible?

Reply:
As in, keep `TestXt` and make it alias to what you declared here.

Reply:
Done.